src/apiCalls.js

```diff
@@ -5,8 +5,7 @@ export const fetchData = () => {
 }
 
 export const fetchSingleData = (id) => {
-    console.log(id.id)
-    return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id.id}`)
+    console.log(id)
+    return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
```

Comment:
Nice job keeping this fetch dynamic.  I would be sure to add some logic to check if the response is ok (on all fetch calls) and if not, throw an Error()